azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1

```diff
@@ -0,0 +1,354 @@
+Start-Transcript -Path C:\Temp\LogonScript.log
+
+## Deploy AKS EE
+
+# Parameters
+$AksEdgeRemoteDeployVersion = "1.0.230221.1000"
+$schemaVersion = "1.1"
+$schemaVersionAksEdgeConfig = "1.5"
```

Comment:
old scheme. update in the base scenario as well

Reply:
changed and tested in base scenario too